modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/ScalaFinchServerCodegen.java

```diff
@@ -89,7 +89,7 @@ public ScalaFinchServerCodegen() {
                         "native")
         );
 
-        defaultIncludes = new HashSet<String>(
+        defaultIncludes = new HashSet<>(
```

Comment:
replace `Sets.newHashSet(...`

Reply:
If you mean the guava Sets.newHashSet() thats considered deprecated since java 7:

https://guava.dev/releases/22.0/api/docs/com/google/common/collect/Sets.html#newHashSet--

> Note for Java 7 and later: this method is now unnecessary and should be treated as deprecated. Instead, use the HashSet constructor directly, taking advantage of the new "diamond" syntax.

Reply:
The one you linked is deprecated, but that's not the one I mentioned above, see `(...`

I meant the method one blow you linked with signature `public static <E> HashSet<E> newHashSet(E... elements)` that is not deprecated.

Reply:
Ack.

Reply:
Done.